pkg/bpm/bpm.go

```diff
@@ -148,8 +148,7 @@ func (m *BackgroundProcessManager) StartProcess(cmd *ManagedProcess) (*process.P
 	go func() {
 		err := cmd.Wait()
 		if err != nil {
-			err, ok := err.(*exec.ExitError)
-			if ok {
+			if err, ok := err.(*exec.ExitError); ok {
```

Comment:
I think we should change the name of `err` in `if err, ok := err.(*exec.ExitError); ok `, this re-assignment of `err` still affects the next `else` branch

Reply:
PTAL @YangKeao

Reply:
Fixed.